.env.test

```diff
@@ -19,4 +19,5 @@ DATABASE_URL="postgresql://user:password@localhost:5432/notifications?schema=pub
 NEXT_PUBLIC_PAWN_SHOP_API_URL="https://nft-pawn-shop-rinkeby.vercel.app"
 NEXT_PUBLIC_NOTIFICATIONS_FREQUENCY_HOURS=1
 EVENTS_API_SECRET_KEY="very secret key"
-BACKED_NOTIFICATIONS_EMAIL_ADDRESS="no-reply@backed.xyz"
\ No newline at end of file
+BACKED_NOTIFICATIONS_EMAIL_ADDRESS="no-reply@backed.xyz"
+EMAIL_MONITOR_REPORT_TO="adamgobes@gmail.com"
```

Comment:
@adamgobes are you sure you want this public? maybe make this something dummy?

Reply:
probably good to make it dummy, but it is also in the log for every commit adam's made

Reply:
will do in a follow up